tests/cross_dock/test_views.py

```
@@ -0,0 +1,230 @@
+class TestFileCleanup:
+    """Tests for file cleanup after processing."""
+
+    @mock.patch("cross_dock.views.os.remove")
+    def test_removes_temporary_file_after_processing(self, mock_remove):
+        """Test that temporary files are removed after processing."""
+        file_path = "/mock/path/test_file.xlsx"
+
+        try:
+            os.remove(file_path)
+        except Exception:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)